src/components/Report.tsx

```diff
@@ -1,58 +1,74 @@
-import React, { useState } from "react";
+/* eslint-disable react/no-unstable-nested-components */
+import React, { useState, useRef } from "react";
 import DatePicker from "react-datepicker";
 import "react-datepicker/dist/react-datepicker.css";
 import { BsCalendarEvent } from "react-icons/bs";
 import "../styles/Report.css";
+import ReactToPrint from "react-to-print";
+import Button from "@mui/material/Button";
 
 export default function Report() {
+  const componentRef = useRef(null);
+
   const [startDate, setStartDate] = useState<Date | null>(null);
   const [endDate, setEndDate] = useState<Date | null>(null);
+
   return (
     <div className="report_box">
-      <span className="intro-text"> Enter Dates </span>
-      <span id="icon">
-        <BsCalendarEvent />{" "}
-      </span>
-      <div id="segment">
-        <div id="divTable" className="InsideContent">
-          <table id="logtable">
+      <div ref={componentRef}>
+        <span className="intro-text"> Enter Dates </span>
+        <span id="icon">
+          <BsCalendarEvent />{" "}
+        </span>
+        <ReactToPrint
```

Comment:
lets add a printer start icon. You can add a prop called startIcon={} and then import and use the printer icon from mui. https://mui.com/material-ui/material-icons/?query=print